build/hellobuild.md

```diff
@@ -0,0 +1,140 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+{% endhighlight %}
+
+Don’t worry about understanding the full example if you’re not familiar with Python - it’s just a simple web server that will contain a single page that says “Hello World”.
+
+> **Note** 
+>
+> If you test the example, make sure to copy over the indentation as well!
+
+**Sample B**
+<br/>Here’s a Dockerfile that Docker Build can use to create an image for our application:
+
+{% highlight dockerfile lineos %}
+# syntax=docker/dockerfile:1
+FROM ubuntu:22.04
+
+# install app dependencies
+RUN apt-get update && apt-get install -y python3 python3-pip
+RUN pip install flask
+
+# install app
+COPY hello.py /
+
+# final configuration
+ENV FLASK_APP=hello
+    EXPOSE 8000
+CMD flask run --host 0.0.0.0 --port 8000
+{% endhighlight %}
+
+* `# syntax=docker/dockerfile:1` 
+
+    This is our syntax directive. It pins the exact version of the dockerfile syntax we’re using. As a [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/), this should be the very first line in all our Dockerfiles as it informs Buildkit the right version of the Dockerfile to use. 
+    <br/>See also [Syntax](../engine/reference/builder/#syntax).
+    
+    > **Note** 
+    > 
+    > Initiated by a "#" like regular comments, this line is treated as a directive when you are using BuildKit (default), otherwise it is ignored.
+
+
+* `FROM ubuntu:22.04`
+
+    Here the `FROM` instruction sets our base image to the 22.04 release of Ubuntu. All following instructions are executed on this base image, in this case, a Ubuntu environment. 
+    The notation `ubuntu:22:04`, follows the `name:tag` standard for naming docker images. 
+    When you build your image you use this notation to name your images and use it to specify any existing docker image.
+    There are many public images you can leverage in your projects. <br/>Explore [Docker Hub](https://hub.docker.com/_/ubuntu/?tab=tags) to find out.
+
+* `# install app dependencies`
+
+    Comments in dockerfiles begin with the # symbol. 
+    As your Dockerfile evolves, comments can be instrumental to document how your dockerfile works for any future readers and editors of the file. 
+    <br/>See also the [FROM instruction](../engine/reference/builder/#from) page in the Dockerfile reference.
+
+* `RUN apt-get update && apt-get install -y python3 python3-pip`
+
+    This `RUN` instruction executes a shell command in the build context. A build's context is the set of files located in the specified PATH or URL. In this example, our context is a full Ubuntu operating system, so we have access to its package manager, apt. The provided commands update our package lists and then, after that succeeds, installs python3 and pip, the package manager for Python.<br/>See also the [RUN instruction](../engine/reference/builder/#run) page in the Dockerfile reference.
```

Comment:
I don't think it's clear in the above where the `PATH` and `URL` come from. I'm also not sure about using the phrase buildx context to describe what's going on here is right - I think we try and use that for the inputs to the build. @crazy-max can correct me if I'm wrong.